modules/check_in/app/workers/check_in/travel_claim_submission_worker.rb

```diff
@@ -5,9 +5,67 @@
 module CheckIn
   class TravelClaimSubmissionWorker
     include Sidekiq::Worker
-    include Sidekiq::MonitoredWorker
+    include SentryLogging
+
     sidekiq_options retry: false
 
-    def perform(uuid, appointment_date); end
+    SUCCESS_TEMPLATE_ID = Settings.vanotify.services.check_in.template_id.claim_submission_success_text
+    DUPLICATE_TEMPLATE_ID = Settings.vanotify.services.check_in.template_id.claim_submission_duplicate_text
+    ERROR_TEMPLATE_ID = Settings.vanotify.services.check_in.template_id.claim_submission_error_text
+
+    SMS_SENDER_ID = Settings.vanotify.services.check_in.sms_sender_id
+
+    STATSD_ERROR_NAME = 'worker.checkin.travel_claim.error'
+    STATSD_SUCCESS_NAME = 'worker.checkin.travel_claim.success'
+
+    def perform(uuid, appointment_date)
+      response = TravelClaim::Response
```

Comment:
Pls check, doesn't seem to be used anywhere

Reply:
It was used in getting `CODE_SUCCESS` but I've removed the reference and used it directly in 6cbda37e4858a7bab666134dc4402a0082c35407